src/main/java/frc/robot/commands/SwitchCamera.java

```diff
@@ -0,0 +1,46 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.command.Command;
+import frc.robot.Robot;
+
+public class SwitchCamera extends Command {
+  public SwitchCamera() {
+    // Use requires() here to declare subsystem dependencies
+    // eg. requires(chassis);
```

Comment:
Need to consider if this is necessary or wanted.

Reply:
Do you mean the requires()?

Reply:
Yes. We don't need to fix it now but eventually we need to figure out whether we want it or not.